src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -159,6 +158,20 @@
                       (error-handler error)
                       (log/error :write-file-failed error)))))))))
 
+(defn- get-file-path [dir path]
+  (let [[dir path'] (map #(-> (when %
+                                (string/replace % "///" "/"))
+                              js/decodeURI)
+                         [dir path])
+        path (if (string/starts-with? path' dir)
+               path
+               (-> (str dir path)
+                   (string/replace "//" "/")))]
+    (js/console.log "cap/write-file!/path: " (js/encodeURI (js/decodeURI path)))
```

Comment:
If the log is needed, better move it into the `if` block beblow

Reply:
No need, remove it now.